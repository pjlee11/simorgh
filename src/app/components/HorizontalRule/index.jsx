import styled from 'styled-components';
import { C_POSTBOX, GEL_SPACING_DBL } from '../../lib/constants/styles';

const HorizontalRule = styled.hr`
  background-color: ${C_POSTBOX};
  height: 0.25rem;
  width: 3rem;
  margin: ${GEL_SPACING_DBL} 0;
  border: 0;
`;

export default HorizontalRule;
