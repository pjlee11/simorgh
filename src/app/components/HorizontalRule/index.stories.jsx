import React from 'react';
import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
import HorizontalRule from './index';
import Text from '../Text';

storiesOf('HorizontalRule', module)
  .add('default', () => <HorizontalRule />)
  .add('between text blocks', () => (
    <React.Fragment>
      <Text text="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc cursus velit vitae interdum euismod. Donec venenatis massa sit amet bibendum accumsan. Mauris hendrerit ullamcorper tortor, vitae vestibulum ipsum vehicula nec. Sed consequat sapien urna, ut porttitor magna condimentum sit amet. Phasellus ullamcorper, lorem non dictum convallis, justo elit pharetra est, sed volutpat justo orci sit amet quam. Nam a mollis ante." />
      <HorizontalRule />
      <Text text="Aenean sed dictum ex. Etiam varius tellus neque, id interdum turpis tempus quis. Aenean in eros ut est sagittis rhoncus ac id metus. Maecenas faucibus urna urna, in placerat nisi accumsan non. Maecenas et auctor sem, non dignissim justo. Quisque nisl mi, malesuada vitae leo vitae, sagittis vulputate urna." />
    </React.Fragment>
  ));
