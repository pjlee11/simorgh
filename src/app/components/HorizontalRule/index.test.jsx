import React from 'react';
import HorizontalRule from './index';
import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';

describe('Headline', () => {
  shouldMatchSnapshot('should render correctly', <HorizontalRule />);
});
