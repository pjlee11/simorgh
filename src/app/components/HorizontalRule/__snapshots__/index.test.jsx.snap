// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should render correctly 1`] = `
.c0 {
  background-color: #B80000;
  height: 0.25rem;
  width: 3rem;
  margin: 1rem 0;
  border: 0;
}

<hr
  className="c0"
/>
`;
